frontend/src/metabase/nav/containers/MainNavbar/MainNavbarView.tsx

```diff
@@ -82,9 +84,22 @@ function MainNavbarView({
     }
   }, [handleCloseNavbar]);
 
+  const isHomePage = window.location.pathname === "/";
+
   return (
     <SidebarContentRoot>
       <div>
+        <SidebarSection>
+          <HomePageLink
+            isSelected={isHomePage}
+            icon="home"
+            onClick={onItemSelect}
+            url="/"
+          >
```

Comment:
`HomePageLink` is a `SidebarLink`, so it uses the`li` HTML tag. Can we wrap it with `ul` to get valid HTML?